README.md

```diff
@@ -124,6 +124,56 @@ spec := &iso8583.MessageSpec{
 }
 ```
 
+The following example creates a full specification with three individual fields (excluding MTI and Bitmap). It differs from the example above, by [showing the expandability of the bitmap field](https://github.com/moov-io/iso8583/blob/5b24f23a5a02206d59baa033ed040f79478b6ecb/field/bitmap.go#L94-L109). This is useful for specs that define both a primary and secondary bitmap.
```

Comment:
I added this, because I was a little confused about how the library interacted with a secondary bitmap. Happy to change this around a bit, but I think we should have an example that shows the library does support a secondary bitmap.

I bring this up, because the 1993 spec has the secondary bitmap defined as field 1, which is where the library puts the primary bitmap.